### DISCUSSION TITLE
Laser Burning while Traversing White Space

### DISCUSSION THREAD

**Discussion by CssTechGuy** [arkypita/LaserGRBL#944] [Q&A]
Hello,

I have a SainSmart 3018 CNC that came equipped with a Woodpecker 3.2 Control Board.
The spindle function works great and I have been able to mill some good projects.
I ordered the SainSmart 5500mw laser to broaden my horizons.
Unfortunately, the laser is ruining my attempted projects as it is burning while traversing through the white space.
I am running GRBL V1.1h and no matter what adjustments I make the issue is not resolved.
Any help would be greatly appreciated.
Attached are my GRBL settings.

![GRBL Settings](https://user-images.githubusercontent.com/45219374/78505310-b0708d80-7740-11ea-870d-538d4f70b4e5.jpg)

**Comment by StuartB4** [comment]
Don't you have to switch from Spindle mode to laser mode somehow on the control board.
I've seen it mentioned somewhere. Because in spindle mode the spindle lifts up but still runs, with the laser it needs to turn off when it moves from location to location.

**Comment by CssTechGuy** [comment]
Stuart,

Yes I've seen that myself on certain boards with jumpers.
There are no jumpers on my Woodpecker 3.2 control board.
There is only a push on/off switch button on the Laser Control Board.


On Sun, 5 Apr 2020 at 14:44, Stuart <notifications@github.com> wrote:

> Don't you have to switch from Spindle mode to laser mode somehow on the
> control board.
> I've seen it mentioned somewhere.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/arkypita/LaserGRBL/issues/944#issuecomment-609462972>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKY74LWEXVVXLUWRJ7PVIHTRLDGRFANCNFSM4MBFQ3VQ>
> .
>


-- 
J.R.(Rick) Hughes

**Comment by DrGio018** [comment]
Could be a problem in G-code with M3/M4.
